dotnet/Razorvine.Pyrolite/Pyrolite/Pyrolite.csproj

```diff
@@ -11,6 +11,9 @@
     <Version>4.24.0</Version>
     <AssemblyVersion>4.24.0.0</AssemblyVersion>
     <FileVersion>4.24.0.0</FileVersion>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>false</DelaySign>
+    <AssemblyOriginatorKeyFile>Pyrolite.snk</AssemblyOriginatorKeyFile>
```

Comment:
(for my information) How did you make this `.snk`? Did you just ask VS to generate a new one?

Reply:
> Did you just ask VS to generate a new one?

Yup.